.github/workflows/frameworks.yml

```diff
@@ -1218,12 +1218,32 @@ jobs:
         run: make tests-tensorflow_v2
         shell: bash
 
+      - name: Run unit test for tensorflow_v2
+        shell: bash
+        run: |
+          OPTS=(--cov=bentoml --cov-config=./setup.cfg --cov-report=xml:"tf2.unit.xml" --cov-report term-missing:skip-covered)
+          python -m pytest ./tests/integration/frameworks/test_tensorflow_v2_unit.py "${OPTS[@]}" || ERR=1
+          if [ $ERR -eq 1 ]; then
+            echo "unit tests for tensorflow_v2 failed!"
+            exit 1
+          fi
+
+      - name: Run tensorflow_v2 without eager execution
+        shell: bash
+        run: |
+          OPTS=(--cov=bentoml --cov-config=./setup.cfg --cov-report=xml:"tf2.unit.no_eager.xml" --cov-report term-missing:skip-covered --disable-eager-execution)
+          python -m pytest ./tests/integration/frameworks/test_tensorflow_v2_unit.py "${OPTS[@]}" || ERR=1
+          if [ $ERR -eq 1 ]; then
+            echo "unit tests for tensorflow_v2 failed!"
+            exit 1
+          fi
```

Comment:
We will revisit CI after releasing 1.0. But for now, we will do something like this per framework.

cc @parano @ssheng